analysis_options.yaml

```diff
@@ -0,0 +1,35 @@
+include: package:effective_dart/analysis_options.yaml
+
+analyzer:
+  exclude: [build/**]
+  errors:
+    argument_type_not_assignable: ignore
```

Comment:
Currently set these lint errors to be ignored. We had 320+ problems with our codebase, so I think we can just handle these linting errors one rule at a time.